packages/badge/src/lib/Badge.stories.tsx

```diff
@@ -11,7 +12,7 @@ export default {
 export const _Badge: StoryObj<typeof Badge> = {
   render: (args: BadgeProps) => <Badge {...args} />,
   args: {
-    children: 'This text is a child of the badge',
+    children: <HomeIcon fontSize="large" />,
```

Comment:
I think the icon should be hard coded as the child instead of passed through the args. This will clean up the code snippet and args table.

Reply:
Done